pythia/scripts/eval.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Change the name of the script to `coco_caption_eval.py` and put it under `pythia/scripts/coco/`